### PR TITLE
Update docs on muting CSM issues

### PR DESCRIPTION
**Pull request by janine-c**
### What does this PR do? What is the motivation?

In this PR:
- I've updated the descriptions of where to find the mute button, which not only has moved, but now varies depending on whether the issue contains one or multiple resources
- I've removed the section about the Resource evaluation over time timeline, which has been removed from the UI

### Merge instructions

Merge readiness:
- [ ] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/security/cloud_security_management/review_remediate/mute_issues.md [hunk 1]
```diff
@@ -23,43 +23,40 @@ For example, the CSM Misconfigurations rule ['Block Public Access' feature is en
...
-{{< img src="security/csm/mute_issue.png" alt="The Mute Issue dialog box contains fields for specifying the reason and duration of the mute" style="width:100%;">}}
+{{< img src="security/csm/mute_issue-2.png" alt="The Mute Issue dialog box contains fields for specifying the reason and duration of the mute" style="width:100%;">}}
 
-1. On the misconfiguration, issue, or identity risk side panel, select one or more resources.
-2. Select **Actions** > **Mute for...**.
-3. Select a reason for the mute, for example, a fix is pending, it's a false positive, or it's an accepted risk.
+1. On the misconfiguration, vulnerability, or identity risk side panel, find the triage status dropdown for the resource.
+   - If the side panel is for a single resource, under **Next Steps**, the list is under **Triage**.
+   - If the side panel is for a list of aggregated resources, select one or more resources in the table, then find the list in the **Status** column.
+2. Open the list with the current triage status and click **Muted**. The Mute issue window opens.
```
Comment (neko-dd) [inline content/en/security/cloud_security_management/review_remediate/mute_issues.md:31]:
```suggestion
2. Open the list with the current triage status and click **Muted**. The **Mute issue** window opens.
```

File: content/en/security/cloud_security_management/review_remediate/mute_issues.md [hunk 2]
```diff
@@ -23,43 +23,40 @@ For example, the CSM Misconfigurations rule ['Block Public Access' feature is en
...
+3. Select a reason for the mute; for example, a fix is pending, it's a false positive, or it's an accepted risk.
 4. Enter an optional **Description**.
 5. Select the duration of the mute.
-6. Click **Mute**.
+6. Click **Mute**. The Mute issue window closes.
```
Comment (neko-dd) [inline content/en/security/cloud_security_management/review_remediate/mute_issues.md:35]:
```suggestion
6. Click **Mute**. The **Mute issue** window closes.
```

File: content/en/security/cloud_security_management/review_remediate/mute_issues.md [hunk 3]
```diff
@@ -23,43 +23,40 @@ For example, the CSM Misconfigurations rule ['Block Public Access' feature is en
...
 
-1. On the misconfiguration, issue, or identity risk side panel, select the resources with the muted issue.
-2. Select **Actions** > **Unmute**.
-3. Select a reason for the unmute, for example, there's no pending fix, it was a human error, or it's no longer an accepted risk.
-4. Enter an optional **Description**.
-5. Click **Unmute**.
+1. On the misconfiguration, vulnerability, or identity risk side panel, find the triage status dropdown for the resource.
+   - If the side panel is for a single resource, under **Next Steps**, the list is under **Triage**.
+   - If the side panel is for a list of aggregated resources, select one or more resources in the table, then find the list in the **Status** column.
+2. Click **Muted** to open the dropdown, then select a new triage status. The triage status updates immediately for the selected resource(s).
```
Comment (neko-dd) [inline content/en/security/cloud_security_management/review_remediate/mute_issues.md:44]:
```suggestion
2. Click **Muted** to open the dropdown, then select a new triage status. The triage status updates immediately for the selected resources.
```

(IIRC when people are using machine translation, usage of (s) can mess it up.)

**Review by neko-dd — APPROVED** [review]
Small suggestions

**Comment by lucasdmaley** [comment]
Two things I noticed:
- If you select more than one resource in the table (on the aggregated resource view), the update is made using the setState dropdown
- Under **To view your organization's muted issues**, it's worth mentioning that the explorer is filtered by default to not show muted issues. To see muted issues in the explorer, the user should remove the @wokflow.triage.status;(open OR in-progress) filter -> this can be done by hovering over it and clicking the X. 
- - I'd add this step in before clicking on "Muted" in the table

Overall, looks great - thanks again for picking up on this!

**Comment by lucasdmaley** [comment]
Two things I noticed:
- If you select more than one resource in the table (in the aggregated resources view), the update is made using the setState dropdown
image
- Under **To view your organization's muted issues**, it's worth mentioning that the explorer is filtered by default to not show muted issues. To see muted issues in the explorer, the user should remove the @workflow.triage.status;(open OR in-progress) filter -> this can be done by hovering over it and clicking the X. 
- - I'd add this in before directing the user to click on "Muted" to sort the table, as that's the order they'd have to do it themselves

Overall, looks great - thanks again for catching and updating this!

**Comment by janine-c** [comment]
Thanks for flagging those, @lucasdmaley ! I've made those updates, so if you take another look at the preview page, you should see them there. Let me know how it looks and don't hesitate to let me know if you need any further changes :)

**Comment by janine-c** [comment]
Thanks, @lucasdmaley , great catch! I added the Vulnerabilities explorer - let me know if you think I got the nuance for that right, or if you'd like to work on the phrasing a bit more :) I appreciate your attention to detail here while I'm still learning the nuances of how these explorers work differently!

**Comment by lucasdmaley** [comment]
No worries at all! Phrasing looks right on what you added, I'd just make sure to add "Vulnerabilities" into this line:
> By default, the Misconfigurations and Identity Risks issue explorers hide muted issues.

**Comment by janine-c** [comment]
@lucasdmaley Updated that wording - it originally omitted the Vulnerabilities explorer because it was leading into the part about manually removing those queries, but hopefully this is clearer :)

**Comment by lucasdmaley** [comment]
Ah, that makes sense - this is definitely clearer. Thank you for going back in to add all of these details, looks good to go!

**Review by lucasdmaley — APPROVED** [review]

**Pull request merged**